tests/baker_recipes/units.py

```diff
@@ -0,0 +1,31 @@
+from model_bakery.recipe import Recipe, foreign_key
+
+# Recipes for the `units.Institution` objects
+
+institution = Recipe(
+    "units.Institution",
+    name="Warsaw University of Technology",
+    abbr="WUT",
+)
+
+
+# Recipes for the `units.Institute` objects
+
+institute = Recipe(
+    "units.Institute",
+    name="Faculty of Chemistry",
+    abbr="Chem",
+    institution=foreign_key(institution),
+)
+
+
+# Recipes for the `units.Department` objects
+
+department = Recipe(
+    "units.Department",
+    name="Department of Physical Chemistry",
+    abbr="Phys",
+    institute=foreign_key(institute),
+)
+
+__all__ = ["department", "institute", "institution"]
```

Comment:
Insert extra line before `__all__`.
```suggestion

__all__ = ["degree", "discipline", "domain"]
```